app/services/rapid_ready_for_decision/lighthouse_medication_request_data.rb

```diff
@@ -41,15 +41,28 @@ def description_hash
       def notes_hash
         verbose_notes = (entry['note'] || [])
 
-        { notes: verbose_notes.map { |note| note['text'] } }
+        { 'notes': verbose_notes.map { |note| note['text'] } }
```

Comment:
Are you intending to use strings as keys? If so, update line 38 for consistency.

Reply:
Woops, no. This is vestigial and improvements were lost after a merge conflict. Thanks.

Reply:
Also: AFAICT, this hash literal will still represent the keys as symbols, even when written as a string:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/5004319/164513062-d4aa95f7-242b-4bde-9318-0ece474f9d28.png">

Reply:
it's the `with_indifferent_access` method that permits string-based keying

Reply:
Acknowledged. It will still work. My comment is to encourage consistency, esp since it's within the same block of code.